cognee/infrastructure/databases/graph/memgraph/memgraph_adapter.py

```
@@ -0,0 +1,690 @@
+
+        params = {"node_ids": node_ids}
+
+        return await self.query(query, params)
+
+    async def delete_graph(self):
+        query = """MATCH (node)
+                DETACH DELETE node;"""
+
+        return await self.query(query)
+
+    def serialize_properties(self, properties=dict()):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)